tests/execute/local_subprocess/test_local_subprocess.py

```diff
@@ -105,14 +105,12 @@ def test_local_execute_write_a_lot(os_env: Dict[str, str]) -> None:
         cmd=[
             sys.executable,
             "-c",
-            (
-                "import sys; import time; from datetime import datetime; import os;"
-                "print('e' * {0}, file=sys.stderr);"
-                "print('o' * {0}, file=sys.stdout);"
-                "time.sleep(0.5);"
-                "print('a' * {0}, file=sys.stderr);"
-                "print('b' * {0}, file=sys.stdout);"
-            ).format(count),
+            f"import sys; import time; from datetime import datetime; import os;"
```

Comment:
This too I think 🤔

Reply:
Looks like this file could be modernized to Python 3 as it was not mentioned in #2015 ? :thinking:

Reply:
It does look like 🤔 unless some tests fail in which case we need to exclude it from pyupgrade 👍🏻 But I think it's alright.